routes/index.js

```diff
@@ -1,10 +1,29 @@
 var express = require('express');
 var router = express.Router();
+var json = require('./data.json')
+
+// need 4 elements from the json array in blog.ejs file
+let data = []
```

Comment:
`data` is a bad name for a variable. Always name variables according to their purpose. In this case `blogs` is better.